src/NServiceBus.RabbitMQ/RabbitMQTransportInfrastructure.cs

```diff
@@ -32,7 +33,19 @@ public RabbitMQTransportInfrastructure(SettingsHolder settings, string connectio
 
             routingTopology = CreateRoutingTopology();
 
-            settings.Set(SettingsKeys.RoutingTopologySupportsDelayedDelivery, routingTopology is ISupportDelayedDelivery);
+            var routingTopologySupportsDelayedDelivery = routingTopology is ISupportDelayedDelivery;
+            settings.Set(SettingsKeys.RoutingTopologySupportsDelayedDelivery, routingTopologySupportsDelayedDelivery);
+
+            if (routingTopologySupportsDelayedDelivery)
+            {
+                var timeoutManagerFeatureDisabled = settings.GetOrDefault<FeatureState>(typeof(TimeoutManager).FullName) == FeatureState.Disabled;
+                var sendOnlyEndpoint = settings.GetOrDefault<bool>("Endpoint.SendOnly");
+
+                if (timeoutManagerFeatureDisabled || sendOnlyEndpoint)
```

Comment:
I don't know much about send-only endpoints, but why is it that we disable the timeout manager in the case of a send-only endpoint?

/cc @Particular/nservicebus-maintainers